components/brave_rewards/browser/rewards_service_browsertest.cc

```diff
@@ -126,10 +129,17 @@ class BraveRewardsBrowserTest : public InProcessBrowserTest,
     } else if (URLMatches(url, WALLET_PROPERTIES, PREFIX_V2,
                           SERVER_TYPES::LEDGER)) {
       GURL gurl(url);
-      if (gurl.has_query())
+      if (gurl.has_query()) {
         *response = brave_test_resp::reconcile_;
-      else
-        *response = brave_test_resp::current_reconcile_;
+      } else {
+        if (ac_low_amount_) {
+          *response = "";
+          *code = 416;
```

Comment:
We should use response status codes from `src/net/http/http_status_code.h`

Reply:
done